### DISCUSSION TITLE
How to create a repository for a new hardware platform

### DISCUSSION THREAD

**Discussion by JustinWu886** [openthread/openthread#9771] [Q&A] [answered]
Hello OpenThread Community,

I am currently working on integrating the OpenThread project into a new Realtek platform and I'm seeking guidance on how to properly create a new repository for this purpose. We have finished the OpenThread porting onto this platform, but we are unsure about the best practices for setting up a repository that is tailored for a new hardware platform. Could someone provide me with a step-by-step guide or point me to the relevant documentation on how to achieve this? Thank you in advance for your assistance! :)

**Comment by jwhui** [comment, marked as answer]
Great to hear that you are looking to contribute support for the Realtek platform!

I'll reach out to you directly about getting a new platform repo set up.

**Reply by JustinWu886** [reply]
Hello Jwhul,

Have you contacted me? I haven't received an email from you. Anyway, I just wanted to let you know that we are seeking your assistance with the new repository for our platform. Thank you in advance.

Best regards,
Justin

**Reply by jwhui** [reply]
Yes, I sent email to the address listed in your GitHub profile on January 17th.

**Reply by JustinWu886** [reply]
I just rechecked my mailbox... your mail is there. I apologize for missing your email... I will reply to you soon.
